ontology/yaml/resources/SAFETY/entity_types/ABSTRACT.yaml

```diff
@@ -79,18 +79,56 @@ FA2X:
   uses:
   - fire_alarm_1
   - fire_alarm_2
+  opt_uses:
+  - primary_fire_alarm
+  - secondary_fire_alarm
     
 WLS:
   id: "944988050314756096"
   description: "Water leak location monitoring."
   is_abstract: true
   uses:
   - water_leak_cable_distance_sensor
-  
+
 LM:
   id: "15773300929664647168"
   description: "Tank level monitoring."
   is_abstract: true
   uses:
   - level_status 
 
+HDS:
+  description: "Standard Hydrogen sensor."
+  is_abstract: true
+  implements:
+  - EQUIPMENT
+  
+WCR:
```

Comment:
Why is this in the safety namespace, and not the PLUMBING namespace?

Reply:
No reason sir, since these are under filters category, I took it to mean they would be under safety. I can move them under plumbing if it's better. Your call sir.